gluster-block-basic.yml

```diff
@@ -0,0 +1,56 @@
+- job:
+    name: gluster-block-basic
+    node: gluster
+    description: Build gluster-block
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        branches:
+        - '*/master'
+        git-tool: Default
+        url: https://github.com/gluster/gluster-block.git
```

Comment:
We don't need it here since the shell script do git clone of this repo.

Reply:
Done